types/hsl-to-rgb-for-reals/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for hsl-to-rgb-for-reals 1.1
+// Project: https://github.com/davidmarkclements/hsl_rgb_converter/
+// Definitions by: jorenbroekema <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Convert hue, saturation and luminosity to rgb
+ *
+ * e.g. hslToRgb(223, 0.44, 0.56) to [93, 121, 192]
+ */
+declare function hslToRgb(hue: number, saturation: number, lightness: number): [number, number, number];
```

Comment:
```suggestion
declare function hslToRgb(hue: number, saturation: number, lightness: number): [red: number, green: number, blue: number];
```
you can safely add named tuples (and this make sense for RGB), as TS4.0 is now minimal version support in DT ecosystem:
https://github.com/DefinitelyTyped/DefinitelyTyped#support-window

Reply:
Ah cool, that's better yeah